src/index.js

```diff
@@ -3,4 +3,4 @@ import ReactDOM from 'react-dom';
 import Table from './Table/Table';
 
 
-ReactDOM.render(<Table />, document.getElementById('root'));
\ No newline at end of file
+ReactDOM.render( < Table / > , document.getElementById('root'));
```

Comment:
🤦🏼

Reply:
what the issue it is forbidden to render table directly